components/sections/Navbar/index.tsx

```diff
@@ -0,0 +1,77 @@
+import Image from 'next/image';
+import type { FC, ComponentProps } from 'react';
+
+import LanguageDropdown from '@/components/Common/LanguageDropDown';
+import ThemeToggle from '@/components/Common/ThemeToggle';
+import NavItem from '@/components/sections/NavItem';
+
+import style from './index.module.css';
+
+type NavItem = { text: string; href: string };
+
+type NavbarProps = {
+  navItems: NavItem[];
+  languages: ComponentProps<typeof LanguageDropdown>;
+  onThemeTogglerClick: () => void;
+};
+
+const Navbar: FC<NavbarProps> = ({
```

Comment:
```suggestion
const NavigationBar: FC<NavbarProps> = ({
```

I feel if wename this NavigationBar it feels more meaningful (not to mention we should fix the casing)

Reply:
Should rename the filename roo, right?

Reply:
Right